contracts/Dependencies/Mynt/MyntLib.sol

```diff
@@ -47,4 +48,71 @@ library MyntLib {
         );
         return _myntMassetManager.redeemTo(_toToken, _dllrAmount, msg.sender);
     }
+
+    /**
+     * @notice Convert DLLR _dllrAmount to _toToken utilizing EIP-2612 permit
+     * to reduce the additional sending transaction for doing the approval to the spender.
+     *
+     * @param _myntMassetManager Mynt protocol MassetManager contract address - needed for integration
+     * @param _toToken bAsset token address to withdraw from DLLR
+     * @param _permit permit data, in form of PermitTransferFrom struct.
+     * @param _permit2 permit2 contract address
```

Comment:
I'd prefer to keep it as param, otherwise, this will affect the whole unit test, as in the canonical Permit2 address actually does not exist in the local test